src/AutoRest.CSharp/Common/Generation/Writers/SerializationWriter.cs

```diff
@@ -83,10 +83,14 @@ private void WriteObjectSerialization(CodeWriter writer, SerializableObjectType
                 }
 
                 writer.Append($"{declaration.Accessibility} partial {(model.IsStruct ? "struct" : "class")} {declaration.Name}")
-                    .AppendRawIf(" : ", model.IncludeSerializer);
-                foreach (var i in serialization.Interfaces)
+                    .AppendRawIf(" : ", model.IncludeSerializer && serialization.Interfaces?.Count() > 0);
```

Comment:
So we should no longer need the check of `IncludeSerializer`.
In our implementation, if this is false, the interface.Count should be zero, therefore we could simplify here:
```suggestion
                    .AppendRawIf(" : ", serialization.Interfaces?.Any() ?? false);
```
`Any` should be faster than Count.